pkg/networking/networking.go

```diff
@@ -96,20 +97,19 @@ func LogResponse(response *http.Response, logger *zerolog.Logger) {
 type defaultHeadersRoundTripper struct {
 	encapsulatedRoundTripper http.RoundTripper
 	networkAccess            *networkImpl
+	logLevel                 zerolog.Level
```

Comment:
Suggestion: please assign the new member in Clone() as well and extend the associated test. Thank you!

Reply:
Which Clone() do you mean? This struct does not have a Clone() func.